packages/contracts-bedrock/contracts/L1/L1StandardBridge.sol

```diff
@@ -142,6 +180,36 @@ contract L1StandardBridge is StandardBridge, Semver {
         _initiateETHDeposit(msg.sender, _to, _minGasLimit, _extraData);
     }
 
+
+    function depositBIT(
+        uint256 _amount,
+        uint32 _minGasLimit,
+        bytes calldata _extraData
+    ) external virtual onlyEOA {
+        _initiateBITDeposit(
+            msg.sender,
+            msg.sender,
+            _amount,
+            _minGasLimit,
+            _extraData
+        );
+    }
+
+    function depositBITTo(
```

Comment:
Please reject bit deposit in `depositERC20` and `depositERC20To`

Reply:
yeah,I will add some condition judge.